scripts/create_baseline_stubs.py

```diff
@@ -46,7 +46,9 @@ def get_installed_package_info(project: str) -> tuple[str, str] | None:
 
     Return (normalized project name, installed version) if successful.
     """
-    r = subprocess.run(["pip", "freeze"], capture_output=True, text=True, check=True)
+    # Not using "uv pip freeze" because if this is run from a global Python,
+    # it'll mistakenly list the .venv's packages.
+    r = subprocess.run(("pip", "freeze"), capture_output=True, text=True, check=True)
```

Comment:
Nit: I find a list slightly more readable/pythonic than a tuple.

Reply:
Yeah let's be consistent with what's already there. Even if my natural instinct is to use a tuple if I don't need to mutate. ^^